airbyte-integrations/connector-templates/source-singer/source_{{snakeCase name}}_singer/source.py.hbs

```diff
@@ -22,20 +22,46 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 """
 
+import json
 
 from airbyte_protocol import AirbyteConnectionStatus
 from base_python import AirbyteLogger
-from base_singer import SingerSource
+from base_singer import SingerSource, Status
 
 
 class Source{{properCase name}}Singer(SingerSource):
+    TAP_CMD = "tap-{{properCase name}}"
```

Comment:
so this assumes that the name of the source is the same of the tap minus the tap? that seems a little jenky. would it be possible to just add another input in the templater where if they select singer, we ask for the name of the singer tap that they will be using?

Reply:
I think this might change with the new singer source class